### ISSUE TITLE
Network dialogs' form layouts are not accessible

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#11156]
The forms in the dialogs on the Network page are not accessible. A big part of the problem is the lack of the `for=""` attribute.

Also, as the dialogs use a _lot_ of dropdowns, our custom ones get in the way, as they are not accessible at all. A solution would be to also port our dropdowns to native HTML5 selects. I have a work-in-progress PR for the native selects @ #11155.

**Comment by KKoukiou** [comment]
Reopened in order to make sure that we address the rest of issues that garrett mentioned here.
